resources/definitions/cremaker_common.def.json

```diff
@@ -0,0 +1,59 @@
+{
+  "version": 2,
+  "name": "cremaker common",
+  "inherits": "fdmprinter",
+  "metadata": {
+    "visible": false,
+    "author": "Joyplace",
+    "manufacturer": "JOYPLACE CO., LTD.",
+    "file_formats": "text/x-gcode",
+    "icon": "icon_ultimaker2",
+    "has_materials": true,
+    "machine_extruder_trains": {
+      "0": "cremaker_extruder_0"
+    }
+  },
+
+  "overrides": {    
+    "machine_heated_bed": { "default_value": true },
+    "machine_center_is_zero": { "default_value": false },
```

Comment:
This is already inherited as `false` from the `fdmprinter.def.json` so no need to include it here.
```suggestion
```